grunt/watch.js

```diff
@@ -1,60 +1,36 @@
+"use strict";
+
 module.exports = {
-    options:
-    {
-        atBegin: true,
```

Comment:
Is it difficult to run `grunt build watch` instead? I thought the point to not running `npm start` on a project was because it ran the installs every time. npm/bower installs have been removed from `npm start` and now it only runs `grunt build connect:local watch`, eg: build, start server, watch